charts/netbox/ci/extras-values.yaml

```diff
@@ -0,0 +1,30 @@
+extraConfig:
+  - values:
+      DUNGEON: synth
+      QUEEN: "will rock you"
+  - configMap:
+      name: extraconfig-cm
+      items: []
+      optional: true
+  - secret:
+      name: extraconfig-secret
+      secretName: "not so secret"
```

Comment:
The indentation was wrong, and I fixed in a previous commit.
Now the actual values/properties are used, and it fails: the mentioned configMap and secret must exist to be referenced and used.
To do this test, we'd need to define extra template with the named configMap and secret.

See 
https://github.com/netbox-community/netbox-chart/blob/a13212749e270a75abe83e55f118bfef8fd361a2/charts/netbox/templates/_helpers.tpl#L123-L129

The templating can probably be improved.

Reply:
Yeah, after figuring out what was wrong in my chart, I'm inclined to just drop this for now. Rye indentation was prettier being a jerk I think?

We should probably have a big values yaml that exercises as much of the chart as possible, given how much time it adds.